roles/neutron-common/templates/etc/serverspec/neutron_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'spec_helper'
+
+describe file('/etc/neutron/rootwrap.d') do
+  it { should be_owned_by 'root' }
+  it { should be_grouped_into 'root' }
+  it { should be_writable.by('owner') }
+end    
+
+describe file('/opt/openstack/current/neutron/etc/neutron/rootwrap.d') do
+  it { should be_owned_by 'root' }
+  it { should be_grouped_into 'root' }
+  it { should be_writable.by('owner') }
+end  
+
+files = ['bin', 'sbin']
+files.each do |file|
+  describe file("/#{file}") do
+    it { should be_owned_by 'root' }
+    it { should be_grouped_into 'root' }
+    it { should be_writable.by('owner') }
+  end 
+end 
+
+files = ['bin', 'sbin']
+files.each do |file|
+  describe file("/usr/#{file}") do
+    it { should be_owned_by 'root' }
+    it { should be_grouped_into 'root' }
+    it { should be_writable.by('owner') }
+  end 
+end 
```

Comment:
Are these literally checking `/bin, /sbin, /usr/bin, /usr/sbin`? How is that a Neutron specific tech spec?